src/quality-patches/release-notes.md

```diff
@@ -0,0 +1,31 @@
+---
+group: software-update-guide
+title: Magento Quality Patches release notes
+functional_areas:
+  - Setup
+  - Configuration
+  - Upgrade
+---
+
+The [Magento Quality Patches](https://github.com/magento/quality-patches) package delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
+
+<!-- The release notes include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+-  {:.bug}Known issues -->
+
+{:.bs-callout-info}
+See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching.html) for instructions on applying patches to your Magento projects.
+
+## v1.0.2
```

Comment:
We found some mistakes in patches configuration, so today we will release v1.0.3. This version will not include new patches, only configuration correction for existing
```suggestion
## v1.0.3
```